spec/rubocop/cop/rails/validation_spec.rb

```diff
@@ -1,225 +1,179 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::Rails::Validation, :config do
-  it 'accepts new style validations' do
```

Comment:
I removed this test, `expect_correction` would raise an `InfiniteLoopError` if this wouldn't pass